src/admin/views/activities/tmpl/default.php

```diff
@@ -15,7 +15,7 @@
 $listOrder     = $this->escape($this->state->get('list.ordering'));
 $listDirn      = $this->escape($this->state->get('list.direction'));
 ?>
-<form action="index.php?option=com_activitystream&view=activities" method="post" id="adminForm" name="adminForm">
+<form action="<?php echo JRoute::_('index.php?option=com_activitystream&view=activities&client=' . $this->client); ?>"  method="post" id="adminForm" name="adminForm">
```

Comment:
@Mayaank12 
no need to route URL in backend

Reply:
Done